secrets.go

```diff
@@ -64,18 +65,45 @@ func makeDefaultHash() HashCfg {
 // types and interfaces
 // ---------------------------------------------------------------------------
 
+type PlainConcealer interface {
+	PlainStringer
+	Concealer
+}
+
+// PlainStringer is the opposite of conceal.
+// Useful for if you want to retrieve the raw value of a secret.
+type PlainStringer interface {
+	PlainString() string
+}
```

Comment:
Just throwing it out there as an option. I don't have any strong opinions.
```suggestion
type Revealer interface {
	Reveal() string
}
```

Reply:
Hah, funny thing, that's the first name I used for it, too!  Alas, while it's fun, I wanted this one to be very clear in its intent and usage.  "Plain string" or "plain text" has a certain emphasis in security that "Reveal" wont.